### ISSUE TITLE
Data-bound shorthand-if with <= breaks extraction

### ISSUE THREAD

**Issue by rubenswieringa** [rubenv/angular-gettext-tools#108]
**tl;dr:** The following breaks string-extraction: `<p translate>{{ n <= 1 ? 'a' : 'b' }}</p>`

# Steps to reproduction

1. In an html-element with a `translate`-attribute, use a smaller-than-or-equal-to operator (`<=`) in a data-bounds (`{{ }}`) expression.
2. Run the `nggettext_extract` grunt-task

**Expected:**

* Only the inner contents of the html-element should be extracted by the grunt-task.

**Actual:**

* Everything after the opening tag of the html-element is extracted (the extracted string might look something like `{{ n <= 1 ? 'a' : 'b' }}</p></div></section>`)

# Tests/code

```html
<!-- test/fixtures/shorthand_if.html -->
<html>
    <body>
        <p translate>1 start {{ n > 1 ? 'if' : 'else' }} end</p>
        <!-- comment after 1st line -->
        <span>after first</span>
        <p translate>2 start {{ n >= 1 ? 'if' : 'else' }} end</p>
        <!-- comment after 2nd line -->
        <span>after second</span>
        <p translate>3 start {{ n < 1 ? 'if' : 'else' }} end</p>
        <!-- comment after 3rd line -->
        <span>after third</span>
        <p translate>4 start {{ n <= 1 ? 'if' : 'else' }} end</p>
        <!-- comment after 4th line -->
        <span>after fourth</span>
    </body>
</html>
```

```javascript
// test/extract_shorthand_if.js

var assert = require('assert');
var fs = require('fs');
var Extractor = require('..').Extractor;
var testExtract = require('./utils').testExtract;

describe('Extract ali baba', function () {
    it('forty thieves', function () {
        var files = [
            'test/fixtures/shorthand_if.html'
        ];

        var paragraphs = [
            "1 start {{ n > 1 ? 'if' : 'else' }} end",
            "2 start {{ n >= 1 ? 'if' : 'else' }} end",
            "3 start {{ n < 1 ? 'if' : 'else' }} end",
            "4 start {{ n <= 1 ? 'if' : 'else' }} end"
        ];

        var catalog = testExtract(files);
        assert.equal(catalog.items.length, paragraphs.length);

        for ( var i = 0; i < paragraphs.length; i++ ) {
            assert.equal(catalog.items[i].msgid, paragraphs[i]);
            assert.equal(catalog.items[i].msgstr, '');
            assert.deepEqual(catalog.items[i].references, ['test/fixtures/shorthand_if.html:' + ( 3 + i * 3 )]);
        }
    });
});
```

# Workaround

Use a greater-than operator (`>`) and swap your ‘if’ and ‘else’ values.
